packages/atomic/src/components/common/query-correction/auto-correction.spec.ts

```diff
@@ -0,0 +1,86 @@
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {createTestI18n} from '@/vitest-utils/testing-helpers/i18n-utils';
+import {i18n} from 'i18next';
+import {html} from 'lit';
+import {describe, beforeAll, it, expect} from 'vitest';
+import {renderAutoCorrection} from './auto-correction';
+
+describe('#renderAutoCorrection', () => {
+  let i18n: i18n;
+
+  beforeAll(async () => {
+    i18n = await createTestI18n();
+  });
+
+  const renderComponent = async (overrides = {}) => {
```

Comment:
Nit,

```suggestion
  const renderComponent = async (overrides: Partial<AutoCorrectionProps> = {}) => {
```

and export the `AutoCorrectProps` interface in auto-correction.ts

Reply:
I usually keep it simple with {}